src/course-home/outline-tab/alerts/certificate-status-alert/CertificateStatusAlert.jsx

```diff
@@ -81,8 +81,6 @@ function CertificateStatusAlert({ intl, payload }) {
       alertProps.header = intl.formatMessage(certMessages.certStatusDownloadableHeader);
       if (isWebCert) {
```

Comment:
Is there a way when `certStatus === CERT_STATUS_TYPE.DOWNLOADABLE` but the `isWebCert` is False?

I mean do we need this `if` statement?

Reply:
`isWebCert` - True if html web certificates are enabled and there is active web certificate
I thought about it, and rather you are right and the IF statement makes no sense